packages/@okta/vuepress-site/guides/sign-into-spa/section10/angular/getuserinfo.md

```diff
@@ -1,4 +1,4 @@
-User info ("claims") is available using the `getUser` method on the `OktaAuthService`.
+Your code can get the user's profile using the `getUser` method on the `OktaAuthService`.
 
 ```javascript
 import { Component, OnInit } from '@angular/core';
```

Comment:
@aarongranick-okta Can you write up a small example of a component that renders the user's name? I'm thinking of `Welcome back, {name}!`

Reply:
working on it...